src/main/java/seedu/duke/Duke.java

```diff
@@ -0,0 +1,66 @@
+package seedu.duke;
+
+import seedu.duke.commands.Command;
+import seedu.duke.commons.Messages;
+import seedu.duke.exception.DukeException;
+import seedu.duke.parser.Parser;
+import seedu.duke.storage.Storage;
+import seedu.duke.task.TaskList;
+import seedu.duke.ui.UI;
+
+public class Duke {
+    private Storage storage;
+    private TaskList tasks;
+    private UI ui;
+
+    public Duke() {
```

Comment:
It would be better if you could add a comment to explain why you have an extra empty constructor